config/common/csi/daemonset.yaml

```diff
@@ -0,0 +1,172 @@
+          initialDelaySeconds: 5
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 1
+        name: driver
+        ports:
+        - containerPort: 10080
+          name: healthz
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 300m
+            memory: 100M
+          requests:
+            cpu: 300m
+            memory: 100M
+        securityContext:
+          privileged: true
+          runAsUser: 0
+          seLinuxOptions:
+            level: s0
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /var/lib/kubelet/plugins
+          mountPropagation: Bidirectional
+          name: plugins-dir
+        - mountPath: /var/lib/kubelet/pods
+          mountPropagation: Bidirectional
+          name: mountpoint-dir
+        - mountPath: /data
+          mountPropagation: Bidirectional
+          name: dynatrace-oneagent-data-dir
+      - args:
+        - --csi-address=/csi/csi.sock
+        - --kubelet-registration-path=/var/lib/kubelet/plugins/csi.oneagent.dynatrace.com/csi.sock
+        - --health-port=9809
+        command:
+        - csi-node-driver-registrar
+        image: quay.io/dynatrace/dynatrace-operator:snapshot
+        imagePullPolicy: Always
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: healthz
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+        name: registrar
+        ports:
+        - containerPort: 9809
+          name: healthz
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 10m
+            memory: 18M
+          requests:
+            cpu: 10m
+            memory: 18M
+        securityContext:
+          runAsUser: 0
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --csi-address=/csi/csi.sock
+        - --health-port=9898
+        command:
+        - livenessprobe
+        image: quay.io/dynatrace/dynatrace-operator:snapshot
+        imagePullPolicy: Always
+        name: liveness-probe
+        resources:
+          limits:
+            cpu: 5m
+            memory: 18M
+          requests:
+            cpu: 5m
+            memory: 18M
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      serviceAccount: dynatrace-oneagent-csi-driver
```

Comment:
can be removed

Reply:
I think some store or whatever complains if you dont give stuff a serviceAccount
don't want to remove this so close to release :sweat_smile:

Reply:
in code `serviceAccount` also wasn't set, only `serviceAccountName` was set:
```
Spec: corev1.PodSpec{
	Containers: []corev1.Container{
		prepareDriverContainer(operatorImage, resourcesMap),
	 	prepareRegistrarContainer(operatorImage, resourcesMap),
	 	prepareLivenessProbeContainer(operatorImage, resourcesMap),
	},
	ServiceAccountName: DefaultServiceAccountName,
	Volumes:            prepareVolumes(),
	Tolerations:        tolerations,
},
```